Makefile

```diff
@@ -28,14 +28,15 @@ CMAKE_CMD ?= cmake
 
 DEFAULT_SUFFIX = clang-debug
 
-COMPILERS := GCC GCC_I686 GCC_FUZZ CLANG EMCC
+COMPILERS := GCC GCC_I686 GCC_FUZZ CLANG CLANG_I686 EMCC
```

Comment:
Seems unrelated? (I'm fine with it but it seems not-quite-small to my eye)

Reply:
Reverted.